src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/When_customizing_outgoing_messages.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace NServiceBus.Transport.RabbitMQ.AcceptanceTests
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using global::RabbitMQ.Client.Events;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    class When_customizing_outgoing_messages : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_set_value()
```

Comment:
Perhaps this test can be more clearly names?

```suggestion
        public async Task Should_set_native_property_value()
```